src/internal/m365/controller_test.go

```diff
@@ -887,49 +887,76 @@ func (suite *ControllerIntegrationSuite) TestRestoreAndBackup_core() {
 				},
 			},
 		},
-		{
-			name:    "MultipleContactsMultipleFolders",
-			service: path.ExchangeService,
-			collections: []stub.ColInfo{
-				{
-					PathElements: []string{"Work"},
-					Category:     path.ContactsCategory,
-					Items: []stub.ItemInfo{
-						{
-							Name:      "someencodeditemID",
-							Data:      exchMock.ContactBytes("Ghimley"),
-							LookupKey: "Ghimley",
-						},
-						{
-							Name:      "someencodeditemID2",
-							Data:      exchMock.ContactBytes("Irgot"),
-							LookupKey: "Irgot",
-						},
-						{
-							Name:      "someencodeditemID3",
-							Data:      exchMock.ContactBytes("Jannes"),
-							LookupKey: "Jannes",
-						},
-					},
-				},
-				{
-					PathElements: []string{"Personal"},
-					Category:     path.ContactsCategory,
-					Items: []stub.ItemInfo{
-						{
-							Name:      "someencodeditemID4",
-							Data:      exchMock.ContactBytes("Argon"),
-							LookupKey: "Argon",
-						},
-						{
-							Name:      "someencodeditemID5",
-							Data:      exchMock.ContactBytes("Bernard"),
-							LookupKey: "Bernard",
-						},
-					},
-				},
-			},
-		},
+		//{
```

Comment:
what's the plan for these tests?  Delete them?  Fix them later?

Reply:
ideally we'll (at some point) all restores to nested contacts folders. We can then re-enable these tests. I'll add a TODO about that to the code

Reply:
I'm a little worried about expectations there.  Msoft's hierarchy support for contacts, whether its increasing or decreasing, is difficult to track.  It _does_ feel slightly like tags are their long term goal? Same result either way, I suppose.  Something to make clear in the todo, maybe.

Reply:
well, if we're not testing it it will be extra difficult to track changes for (though may result in a bit more noise). At some point we'll need to figure out a long term plan for support for contacts and what that should entail

Reply:
Oh, oh, no, I'm not saying we shouldn't test it.  Just pondering what exactly we want to test.  It's weirdness either way. Ah well, this is a good topic for a sync design discussion sometime.